README.md

```diff
@@ -24,7 +30,7 @@ Downgrade a package, looking in only the A.R.M.:
 $ NOCACHE=1 downgrade foo
 ```
 
-Downgrade a package, looking only in local cache, and favoring `su` over 
+Downgrade a package, looking only in local cache, and favoring `su` over
```

Comment:
Thanks for cleaning up the whitespace -- do you might moving those changes to their own commit?